.github/workflows/verify.yml

```diff
@@ -20,3 +20,22 @@ jobs:
       run: |
         go mod tidy
         git diff --exit-code
+
+  lint_markdown:
+    runs-on: ubuntu-20.04
```

Comment:
This could be `latest` instead of `20.04`.

Reply:
The other two jobs were tagging 20.04, so I decided to stick with that for this one too. Would you prefer I change all of them to `latest` instead?

Reply:
You can try 22.04 or latest. If it doesn't break anything, we can switch. For a newly introduced check we can go with latest.